buildpipeline/pipelines.json

```diff
@@ -28,7 +28,8 @@
           },
           "ReportingParameters": {
             "OperatingSystem": "RedHat 7",
-            "Type": "build/product/"
+            "Type": "build/product/",
+            "Architecture": "x64"
```

Comment:
I'm confident this should get Architecture showing up in Mission Control, since this is a ReportingParameter on the sub-job itself